shared/bigqueryUtil.js

```diff
@@ -575,7 +575,7 @@ class BigQueryUtil {
     async insertRows(datasetId, tableId, rows) {
         const dataset = this.bigqueryClient.dataset(datasetId);
         const table = dataset.table(tableId);
-        await table.insert(rows, { raw: false }).then((data) => {
+        return await table.insert(rows, { raw: false }).then((data) => {
```

Comment:
Doesn't this return a Promise based off the docs [here]( https://googleapis.dev/nodejs/bigquery/latest/Table.html#insert)? The cloud build tests have not completed, so not sure if the unittests for *insertRows* is impacted.

Reply:
Yes, previously the function had no return type though, so this shouldn't break any existing tests unless we specifically check for a Promise<void> I think.

Reply:
The cloud build tests completed successfully, but unittest for insertRows only exists in the integration test [here](https://github.com/GoogleCloudPlatform/datashare-toolkit/blob/f896bdcecda9d281196bc3324ccf98b51a9ebc57/shared/test/bigQueryUtilTest.js#L255), can you add the appropriate tests to cover?